content/cloud/index.md

```diff
@@ -8,18 +8,18 @@ date = "2022-03-14T00:22:56Z"
 - [Getting Support](#getting-support)
 - [Open Beta](#open-beta)
 
-[The Fermyon Cloud](https://cloud.fermyon.com) is a cloud application platform for WebAssembly microservices. It enables you to run [Spin applications](/spin), at scale, in the cloud, without any infrastructure setup.
+[The Fermyon Cloud](https://cloud.fermyon.com) is a cloud application platform for WebAssembly-based serverless functions and microservices. It enables you to run [Spin applications](/spin), at scale, in the cloud, without any infrastructure setup or maintenance required. You can take the same Spin applications you were running locally with `spin up` and push them to Fermyon Cloud with a simple `spin cloud deploy` command. That's just the beginning; let's take a closer look at what you can learn about with the Fermyon Cloud documentation. 
```

Comment:
Do we prefer to refer to `spin deploy` or `spin cloud deploy`?  They are defined to mean the same thing but a lot of our collateral uses `spin deploy` because it's simpler.

Reply:
Thanks Ivan.
I think just `spin deploy` is best here; aligns with the other documentation.